cpp/src/writer/tsfile_writer.cc

```diff
@@ -220,8 +220,7 @@ int TsFileWriter::do_check_schema(const std::string &device_name,
         if (UNLIKELY(ms_iter == msm.end())) {
             chunk_writers.push_back(NULL);
         } else {
-            // Here we may check data_type against ms_iter. But in Java
-            // libtsfile, no check here.
+            // In Java we will check data_type. But in C++, no check here.
```

Comment:
Please give a reason for this.

Reply:
because checks are performed at the chunk layer and page layer

Reply:
Add in the code also, not just the conversation.

Reply:
ok